### DISCUSSION TITLE
Check Mark Color

### DISCUSSION THREAD

**Discussion by Droppisalt** [sct/overseerr#2250] [Feature Requests]
Hello, I'd like to know if there's any chance to add a border (maybe black?) to the check mark icon that says if a tv show/movie is already in the library. I'm colorblind and there's a lot of cases where I can't see it and it makes it hard to scroll through the content. 

Ex.: I can easily see it for "The Bad Batch" but I had to focus on "Visions" to be able to see it. 

I do understand that I'm in the minority and understand if this low priority or outside the scope of your development. 

![image](https://user-images.githubusercontent.com/67336980/139563318-cfb43447-cc5e-4c79-8ee6-a140cfb06f71.png)
![image](https://user-images.githubusercontent.com/67336980/139563288-94c7fc77-19aa-4331-8c4a-0dc7ff6e65e5.png)

**Comment by TheCatLady** [comment]
There was a discussion about this on Discord back in January: https://discord.com/channels/783137440809746482/783137440809746485/795643105491157033

TLDR; there used to be a border, but it looked funky and was removed.

**Reply by Droppisalt** [reply]
ahhh damn.. Oh well. Thanks for the info

**Reply by ghero101** [reply]
I will say I struggled with this for much of the same reason specifically when I was going through a discover queue and seeing a lot of things I already had in my library. 

I will say that the "experimental" feature to hide available media went a long way for me. Since it hides all from what I can tell of things I already have, then the checks become a mute point because they only show on those things you already have. You may want to consider trying out that feature. In the General Tab of settings Hide Available Media.

Just from my experience as fellow who also struggles will color deficiencies.
